src/app/docs-pages/app-shell-doc/app-shell-doc.component.ts

```diff
@@ -0,0 +1,50 @@
+import { Component, OnInit } from '@angular/core';
+import { ApiTableRow } from 'src/app/shared/models/api-table-row.interface';
+
+@Component({
+  selector: 'hal-app-shell-doc',
+  templateUrl: './app-shell-doc.component.html',
+  styleUrls: ['./app-shell-doc.component.scss']
+})
+export class AppShellDocComponent implements OnInit {
+
+  api: ApiTableRow[] = [
+    { apiInput: 'menuItems', description: 'List of the menu elements with url, svg file name, title and potentially groupTitle' },
+    { apiInput: 'user', description: 'The current active user' },
+    { apiInput: 'settingsPagePath', description: 'URL to settings page in your application' },
+    { apiInput: 'signOutPagePath', description: 'URL to sign out page in your application. That page should auto sign out the user' },
+    { apiInput: 'ngProjectAs="pageTitle"', description: 'The current page title' },
+    { apiInput: 'ngProjectAs="pageContent"', description: 'The page content. Either a router-outlet or a component' },
```

Comment:
Vi har skrevet apiInput: '[menuItems]' med klammene i stedet for apiInput: 'menuItems' de fleste steder. Det er kanskje litt tydligere?